bats/fb-content-katello.bats

```diff
@@ -121,8 +121,13 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
+  yum erase -y 'katello-ca-consumer-*' || true
   rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm || true
   subscription-manager register --force --org="Test_Organization" --activationkey="Test AK" || true
+  if tIsRHEL && ! tIsCentOSCompatible; then
+    # skipping the rest of the test as we don't provide RHEL subs
+    skip
+  fi
```

Comment:
Will the line below this fail on RHEL? Is that cause Current won't be present since there will be 0 subs? Even custom product subs?